### PR TITLE
Lets master vampires set the name of their clan

### PR DESCRIPTION
**Pull request by MadmanMartian**
Working towards vampire clans

This lets the 'master' vampire name their clan.

### CODE REVIEW

**Comment by Durgen** [comment]
>Working towards vampire clans

Based

V:TMB?

**Comment by MadmanMartian** [comment]
V:TM, yeah

Current plan - Master vampires at roundstart get to pick clan quirks. Some are positive, some are negative, for every positive you take you have to take a negative.

Once you've got your clan set up, you continue the current escallation that's in place (can be tweaked) and depending on the power level of your clan, you can turn a thrall into a fledgling vampire. This is an expensive and long process, so you can't just become a vampire factory.

A fledgling vampire can't ascend to full vampirehood until he takes a victim, starts making his own thralls, etc. and in doing so forfeits themselves from the clan (making him an enemy of the clan), but also makes his own. This inherits the pros and cons quirks of the previous clan, and allows you to select more and branch off. A 2nd generation vampire can pick from a different pool of quirks than a 1st generation clan would be able to, etc.

**Comment by jknpj** [comment]
ANTI CULT CLAN

File: code/datums/gamemode/role/vampire_role.dm [hunk 1]
```diff
@@ -53,7 +53,9 @@
 			to_chat(antag.current, "Drink blood to gain new powers and use coffins to regenerate your body if injured.")
 			to_chat(antag.current, "You are weak to holy things and starlight.")
 			to_chat(antag.current, "Don't go into space and avoid the Chaplain, the chapel, and especially Holy Water.")
-
+			if(greeting == GREET_MASTER && faction && istype(faction, /datum/faction/vampire) && faction.leader == src)
```
Comment (ShiftyRail) [inline code/datums/gamemode/role/vampire_role.dm]: I would personally put this on `OnPostSetup` rather than `Greet`.

**Comment by ghost** [comment]
I'm waiting for the first 'reich' clan when this gets merged

File: code/datums/gamemode/role/vampire_role.dm [hunk 2]
```diff
@@ -469,7 +471,7 @@
 /datum/role/thrall/Greet(var/you_are = TRUE)
 	var/dat
 	if (you_are)
-		dat = "<span class='danger'>You are a Thrall!</br> You are slaved to <b>[master.antag.current]</b>!</span>"
+		dat = "<span class='danger'>You are a Thrall!</br> You are slaved to <b>[master.antag.current]</b>[faction?"under the [faction.name] clan!":""]</span>"
```
Comment (Intigracy) [inline code/datums/gamemode/role/vampire_role.dm]: nitpicky, but if there's no faction then there's no `!` punctuation

Comment (MadmanMartian) [reply code/datums/gamemode/role/vampire_role.dm]: good spot.

**Comment by Intigracy** [comment]
How many chances do they get to use the name proc?

**Comment by MadmanMartian** [comment]
One?

**Pull request merged**